modin/core/io/file_dispatcher.py

```diff
@@ -213,7 +210,9 @@ def get_path(cls, file_path):
         if `file_path` is an S3 bucket, parameter will be returned as is, otherwise
         absolute path will be returned.
         """
-        if isinstance(file_path, str) and S3_ADDRESS_REGEX.search(file_path):
+        if isinstance(file_path, str) and (
+            fsspec.core.split_protocol(file_path)[0] in ("s3", "S3", "https", "http")
```

Comment:
let's make a module-level constant like `_SUPPORTED_PROTOS = {"s3", "S3", "http", "https"}` and use it everywhere

Reply:
The constants were added for `file_dispatcher` and `csv_glob_dispatcher`